### ISSUE TITLE
Starter Pack

### ISSUE THREAD

**Issue by gcopoix** [mist-devel/mist-binaries#162]
Hi,
first of all great work reg. the hardware and collecting/writing these cores.
The [starter_pack](https://github.com/mist-devel/mist-binaries/tree/master/starter_pack) is quiet outdated (9 years), may be interesting for you to have a look at my [installation scripts](https://github.com/gcopoix/makeSD_mist) supporting MiST, SiDi and SiDi128.
Could be worth a link as this might help people creating an SD card set.
Thank you

**Comment by gyurco** [comment]
Yeah, even the "Getting Started" section in the Wiki is outdated, it would be good to update.
